js-api-spec/value/boolean.test.ts

```diff
@@ -32,6 +32,7 @@ describe('Sass boolean', () => {
       expect(value.tryMap()).toBe(null);
       expect(value.assertNumber).toThrow();
       expect(value.assertString).toThrow();
+      expect(value.assertMixin).toThrow();
```

Comment:
Nit: put `assertMixin` after `tryMap` and before `assertNumber` so that these are ordered alphabetically.